src/js/components/Image/Image.js

```diff
@@ -1,10 +1,22 @@
+import React, { useState } from 'react';
 import { StyledImage } from './StyledImage';
 
+const Image = ({ src, fallback, ...rest }) => {
+  const [imageMissing, setImageMissing] = useState(false);
+  return (
+    <StyledImage
+      {...rest}
+      onError={() => setImageMissing(true)}
```

Comment:
What if the caller passes an `onError` as well, this seems like we will be ignoring it?

Also, it would keep the snapshots cleaner if we only added this when there is a fallback.

Reply:
damn true i didn't think about this. i ll fix this in a subsequent PR